.github/workflows/ci_cd.yml

```diff
@@ -336,15 +351,38 @@ jobs:
           docker load --input /tmp/ingestion_server/ingestion_server.tar
 
       - name: Build docs
-        run: just sphinx-make
+        run: |
+          just sphinx-make
+          sudo chown "$USER:$USER" -R ./api
+          mv ./api/build/html /tmp/preview
+
+      # Otherwise we end up with any extra stuff not git-ignored in the gh-pages branch
+      - name: Recreate working directory to avoid superfluous files
+        run: |
+          cd ..
+          sudo rm -rf openverse-api
+          mkdir openverse-api
+
+      - uses: actions/checkout@v3
+        with:
+          ref: gh-pages
+
+      - name: Merge preview with existing gh-pages
+        run: |
+          cd ..
+          cp -r ./openverse-api /tmp/gh-pages
```

Comment:
if the `openverse-api` folder was removed and recreated in a previous step, what is being copied here?

Reply:
The preceeding step is to re-checkout the `gh-pages` branch. This happens after the directory is removed and recreated:

```yaml
      # Otherwise we end up with any extra stuff not git-ignored in the gh-pages branch
      - name: Recreate working directory to avoid superfluous files
        run: |
          cd ..
          sudo rm -rf openverse-api
          mkdir openverse-api

      - uses: actions/checkout@v3
        with:
          ref: gh-pages
```

So here we're copying the current contents of the `gh-pages` branch.